docs/sphinx/examples/cpp/algorithms/bernstein_vazirani.cpp

```diff
@@ -10,19 +10,19 @@
 // simulated using the multi-GPU backend. The amount of resources required for
 // the simulation doubles with with each additional qubit.
 
-#include <bitset>
 #include <cudaq.h>
 #include <iostream>
 #include <random>
+#include <vector>
 
 #ifndef SIZE
 #define SIZE 5
 #endif
 
 template <int nrOfBits>
-std::bitset<nrOfBits> random_bits(int seed) {
+std::vector<bool> random_bits(int seed) {
 
-  std::bitset<nrOfBits> randomBits;
+  std::vector<bool> randomBitsi(nrOfBits);
```

Comment:
```suggestion
  std::vector<bool> randomBits(nrOfBits);
```
Also - I don't think `std::vector<bool>` supports the `.set` method used in line 30?